src/components/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
```

Comment:
- [ ] The `App` component is a top-level component and document structure-wise it is more logical to put this file in the `src` folder. Kindly make changes here. :pray: